### PR TITLE
[Merged by Bors] - doc: fix diagram formatting

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by grunweg** [comment]
Thanks for going over these! The changes seem plausible --- but I have no way to verify. Is there any way to look at the new documention, for instance, to check that the diagrams render better now?

File: Mathlib/CategoryTheory/Limits/Shapes/Pullback/Pasting.lean [hunk 1]
```diff
@@ -46,11 +46,13 @@ variable (h₁ : i₁ ≫ g₁ = f₁ ≫ i₂) (h₂ : i₂ ≫ g₂ = f₂ ≫
 
 /-- Given
 
+```
 X₁ - f₁ -> X₂ - f₂ -> X₃
```
Comment (eric-wieser) [inline Mathlib/CategoryTheory/Limits/Shapes/Pullback/Pasting.lean:50]: Perhaps worth doing the horizontal arrows too

**Review by eric-wieser — APPROVED** [review]
bors d+

Thanks!

File: Mathlib/CategoryTheory/Limits/Shapes/Pullback/Pasting.lean (hunk 1 above)
Comment (Ruben-VandeVelde) [reply Mathlib/CategoryTheory/Limits/Shapes/Pullback/Pasting.lean:50]: Note that the `f`s are inside the arrow, so I'm not sure whether that'd be more or less readable

**Comment by Ruben-VandeVelde** [comment]
bors merge

**Pull request closed without merging**